Packet++/header/TcpLayer.h

```diff
@@ -6,6 +6,7 @@
 #include <string.h>
 
 #ifndef PCPP_DEPRECATED_TCP_OPTION_TYPE
```

Comment:
we already have `#pragma once`, so maybe we don't need `PCPP_DEPRECATED_TCP_OPTION_TYPE`?

Reply:
Wdym? They are different things?